CDCgov/NEDSS-Modernization
# CNFT1-1710-Patient administrative table component consistency: patient administrative

## Description

Table Component consistency for Administrative Race

## Tickets

* [CNFT1-1710](https://cdc-nbs.atlassian.net/browse/CNFT1-1710)
## Steps to verify
Log in
Navigate to advanced search
Search for any patient
Navigate to patient profile and navigate to Demographics tabs and observe the Administrative table functioning as expected.

## Checklist before requesting a review


[CNFT1-1710]: https://cdc-nbs.atlassian.net/browse/CNFT1-1710?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
